internal/scripts/deploy-dotcom.ts

```diff
@@ -415,6 +399,22 @@ function updateFlyioToml(appName: string): void {
 	fs.writeFileSync(flyioTomlFile, updatedContent, 'utf-8')
 }
 
+async function deployPermissionsToFlyIo() {
+	const schemaPath = path.join(REPO_ROOT, 'packages', 'dotcom-shared', 'src', 'tlaSchema.ts')
+	const permissionsFile = 'permissions.sql'
+	await exec('npx', [
+		'zero-deploy-permissions',
+		'--schema-path',
+		schemaPath,
+		'--output-file',
+		permissionsFile,
+	])
+	const result = await exec('psql', [env.BOTCOM_POSTGRES_CONNECTION_STRING, '-f', permissionsFile])
```

Comment:
The command failed, but it didn't stop the deployment. Added a check for it so that it now fails: https://github.com/tldraw/tldraw/actions/runs/14306723803/job/40092470607?pr=5834#step:12:1017